telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/autocomplete/AutocompleteHandler.java

```diff
@@ -20,11 +20,12 @@
 
     private final BotApi botApi;
     private final Map<String, String[]> autocompletes = new HashMap<>();
+    private final String pattern;
 
     @Autowired
     public AutocompleteHandler(BotApi botApi, AutocompleteConfig config) {
         this.botApi = botApi;
-
+        this.pattern = Pattern.compile("[^\\/\\n]+\\/([^\\/\\n]+\\/)*[^\\/\\n]+").toString();
```

Comment:
.toString()? Why? You should compile and cache pattern in field for further use.